app-frontend/src/app/core/services/geocode.service.js

```diff
@@ -0,0 +1,34 @@
+/* global $ */
+/* eslint-disable camelcase */
+const appId = 'mXP4DZFBZGyBmuZBKNeo';
+const appCode = 'kBWb6Z7ZLcuQanT_RoP60A';
```

Comment:
Maybe one way to handle this would be to allow it to be overidden via the webpack overrides mechanism?